bindings/netstandard/ElectionGuard/ElectionGuard.ElectionSetup/Guardian.cs

```diff
@@ -93,11 +94,11 @@ protected override void DisposeUnmanaged()
 /// </summary>
 public class Guardian : DisposableBase
 {
-    private const string GuardianPrefix = "guardian_";
-    private const string PrivateKeyFolder = "gui_private_keys";
-    private const string GuardianExt = ".json";
+    internal const string GuardianPrefix = "guardian_";
+    internal const string PrivateKeyFolder = "gui_private_keys";
+    internal const string GuardianExt = ".json";

```

Comment:
Since these are only used in the extention now, should they move to the extention instead of being here.